examples/ingest/elasticsearch/destination.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Processes all the files from s3://utic-dev-tech-fixtures/small-pdf-set/,
+# embeds the processed documents, and writes to results to a Pinecone index.
```

Comment:
not pinecone

Reply:
[fix example](https://github.com/Unstructured-IO/unstructured/pull/2152/commits/52b4ac9ba4dff1e73634923a0c98335a7b8326a7)